content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Add a job to testgrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+
+## What is testgrid?
+Testgrid is an aggregator for job results across many runs, allowing us to display test results over time and thus giving an indication of current health, especially in the face of flaky tests.
+
+## What are release-informing/blocking/broken jobs?
+* `Blocking jobs` are those that signal widespread failure of the platform. These are traditionally the core end-to-end test runs on our major platforms and upgrades from previous versions. 
+* `Informing jobs` are a broader suite that test the variety of enviroments and configurations our customers expect. 
+* `Broken jobs` are those that have a known, triaged failure that prevents their function for a sustained period of time (more than a week).
+
+## How to decide if a job should qualify as a release-informing job?
+Release informing jobs in the [release controller config](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases) get run every time we build a new payload for that release.
+So you need to ask yourself whether the given job:
+1) is reliable enough to provide a useful signal when it fails
+2) covers an important function that determines whether the payload is viable or not, which is not covered by other informing or blocking jobs
+
+If it does not meet the qualifications for being a release controller informing job, it can still be added to testgrid (and thus [sippy](https://sippy.ci.openshift.org/)) by adding it to the [_allow-list](https://github.com/openshift/release/blob/master/core-services/testgrid-config-generator/_allow-list.yaml). In which case it won't be run against every new payload that is built, it would instead just run on whatever periodic schedule you configured it for.
+
+## How are jobs classified as release-informing or blocking?
+The [release config](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases) and the `optional` job annotation combine to determine the testgrid dashboard. If a job in the release definition is an upgrade job it goes into the overall informing dashboard (because upgrades cross dashboards). If it is optional it is considered informing, and is otherwise considered blocking. 
```

Comment:
This is explaining how it is done mechanically but not explaining the process by which a developer and pillar lead can choose to make these mechanical changes. Please port over the verbiage from the Google Doc and get the architects to review this